Land based autonomous navigation/Land based autonomous navigation/MDK/MDK.paths.props

```diff
@@ -7,9 +7,9 @@
     -->
     <MDKVersion>1.4.7</MDKVersion>
     <MDKUseGameBinPath>no</MDKUseGameBinPath>
-    <MDKGameBinPath>D:\Steam\SteamApps\common\SpaceEngineers\Bin64</MDKGameBinPath>
-    <MDKInstallPath>c:\users\marco\appdata\local\microsoft\visualstudio\16.0_07400820\extensions\ovksfp5o.qfi</MDKInstallPath>
-    <MDKOutputPath>C:\Users\marco\AppData\Roaming\SpaceEngineers\IngameScripts\local</MDKOutputPath>
+    <MDKGameBinPath>c:\program files (x86)\steam\SteamApps\common\SpaceEngineers\Bin64</MDKGameBinPath>
```

Comment:
I believe we need to ignore this

Reply:
I believe so too..
In a haste I forgot to include a proper .gitignore